code/_onclick/other_mobs.dm

```diff
@@ -1,3 +1,13 @@
+/// Checks for RIGHT_CLICK in modifiers and runs attack_hand_secondary if so. Returns TRUE if normal chain blocked
+/mob/living/proc/right_click_attack(atom/A, list/modifiers)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names